src/parser.pegjs

```diff
@@ -430,6 +470,9 @@ BareCodeBlock
 Code
   = $((![{}] SourceCharacter)+ / "{" Code "}")*
 
+Integer
+  = digits:$DecimalDigit+ { return parseInt(digits); }
```

Comment:
I like to make the `10` explicit in parseInt, but don't bother to fix unless you agree.